faraday/proxy.py

```diff
@@ -581,7 +581,12 @@ def main():
 
     # Associate serial ports with callsigns
     # global units
-    units = callsign2COM()
+    try:
+        units = callsign2COM()
+
+    except ConfigParser.NoSectionError as e:
+        logging.error(e)
+        sys.exit()
```

Comment:
Updated in https://github.com/FaradayRF/Faraday-Software/pull/193/commits/49a24e73af1a7b220b169340dc2ada718ce64fae

Picking status code 1 = configuration error. Where do we think this should be documented? I'm thinking either in `contributions.md` or a separate `errors.md` type file?